requirements.txt

```diff
@@ -1,19 +1,36 @@
 appdirs==1.4.3
```

Comment:
@hellysmile I cannot see where requirements are used. Would you help me to find out plz?

Reply:
it mostly what I am using when doing local development of library
should I remove it at all?